README.md

```diff
@@ -21,65 +16,65 @@ It is a tiny alternative to more capable projects:
 - Representing state in a map:
   [robertluo/fun-map](https://github.com/robertluo/fun-map)
 
-## TL;DR example
-
 ``` clojure
-;; in your project
-(require '[piotr-yuxuan/closeable-map :refer [closeable-map]])
+;; In your project, require:
+(require '[piotr-yuxuan.closeable-map :refer [closeable-map closeable-hash-map] :as system])
 
 (defn start
   "Return a running context with values that can be closed."
   [config]
-  (closeable-map {:server (http/start-server (api context) {:port 3030})
-                  :producer (kafka-producer config)
-                  :consumer (kafka-consumer config)}))
+  (closeable-map
+    {;; Kafka producers/consumers are `java.io.Closeable`.
+     :producer (kafka-producer config)
+     :consumer (kafka-consumer config)
 
-;; in test file
-(with-open [context (start config)]
-  (testing "unit test with isolated, repeatable context"
-    (is (= :yay/🚀 (some-business/function context)))))
-```
+     ;; File streams are `java.io.Closeable` too:
+     :outfile (io/output-stream (io/file "/tmp/outfile.txt"))
 
-Syntactic sugar:
+     ;; Closeable-maps can be nested. Also, you can choose to ignore
+     ;; errors either for the root map, or a nested map:
+     :db ^:closeable-map/ignore-errors {:db-conn (jdbc/get-connection (:db config))}
```

Comment:
As is, should be `^:system/ignore-errors`. Is it a good example for ^::ignore-errors and nested maps? We probably can think about something better.